models/config.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package models
```

Comment:
Moved into `config` package.

The reason why I decided to use the `models` package is that I had to avoid cycle imports.
But I moved the validator to the `config` package and everything is ok.

[Commit c5a053d](https://github.com/conduitio-labs/conduit-connector-oracle/commit/c5a053d5eef8eb8e56b449c456dd9a8fbdf64cfd)